pkg/scheduler/framework/plugins/volumebinding/binder_test.go

```diff
@@ -540,8 +540,8 @@ func (env *testEnv) validateBind(
 		// Cache may be overridden by API object with higher version, compare but ignore resource version.
 		newCachedPV := cachedPV.DeepCopy()
 		newCachedPV.ResourceVersion = pv.ResourceVersion
-		if !reflect.DeepEqual(newCachedPV, pv) {
-			t.Errorf("cached PV check failed [A-expected, B-got]:\n%s", diff.ObjectDiff(pv, cachedPV))
+		if !reflect.DeepEqual(pv, newCachedPV) {
```

Comment:
What's the intention here?

Reply:
The `DeepEqual` func here compared the expectedPV and actualPV, to keep consistent with the Errorf and same compare order on #L436, #L441 and #L457, which all put the expected obj in the first order.

Reply:
better to use `cmp.Diff` instead, like
```
	if diff := cmp.Diff(pv, newCachedPV); diff != "" {
		t.Errorf("unexpected PV (-want, +got):\n%s", diff)
	}
```

Reply:
OK, updated with `cmp.Diff`.
PTAL thanks.